core/lib/zksync_core/src/state_keeper/batch_executor/mod.rs

```diff
@@ -84,6 +84,7 @@ pub struct MainBatchExecutorBuilder {
     save_call_traces: bool,
     max_allowed_tx_gas_limit: U256,
     upload_witness_inputs_to_gcs: bool,
+    enum_index_migration_chunks: u16,
```

Comment:
nit: this reads as the **number of chunks** , not the **chunk_size**. I would rename it to `chunk_size`

Reply:
I actually meant number of chunks, anyway approach will be changed